modules/auth/ldap/ldap.go

```diff
@@ -249,12 +339,12 @@ func (ls *Source) SearchEntry(name, passwd string, directBind bool) *SearchResul
 
 	var isAttributeSSHPublicKeySet = len(strings.TrimSpace(ls.AttributeSSHPublicKey)) > 0
 
-	attribs := []string{ls.AttributeUsername, ls.AttributeName, ls.AttributeSurname, ls.AttributeMail}
+	attribs := []string{ls.AttributeUsername, ls.AttributeName, ls.AttributeSurname, ls.AttributeMail, ls.UserAttributeInGroup}
```

Comment:
Sorry I missed this earlier but if `ls.UserAttributeInGroup` is empty - does ldap tolerate the empty attribute?

If it's empty I would guess that you should just not add it to the attribs to get.

Reply:
```suggestion
	attribs := []string{ls.AttributeUsername, ls.AttributeName, ls.AttributeSurname, ls.AttributeMail}
	if len(strings.TrimSpace(ls.UserAttributeInGroup)) > 0 {
		attribs = append(attribs, ls.UserAttributeInGroup)
	}
```